### DISCUSSION TITLE
Got a new Tesla but Teslamate is showing the old data from the old car

### DISCUSSION THREAD

**Discussion by marcbaier** [teslamate-org/teslamate#3187] [Q&A]
I just got my new Model X and was expecting the data for the new car to just appear in teslamate. However i still see the old data from when I reset the now sold Model S. I restarted the Teslamate Container but I still see the old data. When I click on "settings" in the teslamate interface I have two tabs. One with my old car and one for my new car. How can I tell the Grafana interface to only show that data ?

**Comment by CarlosCuezva** [comment]
Is your old Model S still linked to your Tesla account?

**Comment by marcbaier** [comment]
No it is not. I removed it. On 19 Apr 2023, at 19:14, Carlos Cuezva ***@***.***> wrote:﻿
Is your old Model S still linked to your Tesla account?

**Reply by CarlosCuezva** [reply]
Ok, so you will have to delete the old car from the database.

In this discussion explains how to do it 👇

#3074

**Comment by marcbaier** [comment]
Thanks a lot. Looked at the solution but can’t figure out how to find the car ID that I need to delete. Where can I find that ?

**Reply by cwanja** [reply]
Follow the guide here:

https://github.com/adriankumpf/teslamate/pull/3075/files

It's a pull request, but it is an update to documents.

**Reply by CarlosCuezva** [reply]
```
SELECT id, vin , CONCAT('Model ', model, ' ', marketing_name) AS model FROM cars;
```

In your case, in the "model" field will appear "Model S ..."

This is an example of the query with my data

![Captura de pantalla 2023-04-20 a las 6 33 30](https://user-images.githubusercontent.com/5815421/233259096-10fef89c-30f5-4bce-8b56-3c8583145240.png)

**Reply by marcbaier** [reply]
I am very sorry but I am not a programmer and it is not clear to me what I need to do where. Do I need to access the database on my Synology NAS docker container via SSH ? If yes, can I do that using the admin user ? If yes, then where exactly do I need to go ? I also tried to just just delete my existing containers and do a total reinstall using "docker-compose up -d" but grafana still shows the old data. I was expecting that a deletion and reinstall would fix the issue ? Maybe I did not delete everything ?

In the teslamate setting I also still see this weird "???" Entry which confuses me. 

Any help would be greatly appreciated and sorry for the many questions. 
<img width="1077" alt="Screen Shot 2023-04-20 at 09 11 50 AM" src="https://user-images.githubusercontent.com/97667985/233288502-344e983a-7f47-4822-ac88-d1a24195aa96.png">

**Reply by marcbaier** [reply]
OK I now deleted all the containers as well as images and now I am seeing both cars in the grafana dashboard.
So I guess I still somehow need to go in there and manually delete old data. 

<img width="1197" alt="Screen Shot 2023-04-20 at 09 29 12 AM" src="https://user-images.githubusercontent.com/97667985/233292451-3174a351-8c7f-4722-9131-ff190958c80e.png">
